attestation.md

```diff
@@ -0,0 +1,89 @@
+# 🔖 Attestation 
+
+In this section, you'll play the role of an <span class="label-role attester">attester</span>:
+* You'll take a `RequestForAttestation` object; 
+* Attest it;
+* Store the attestation on the chain;
+* Build the `AttestedClaim` object, which will be send back to the <span class="label-role claimer">claimer</span>.
+
+## Get some tokens as an attester  
+
+You'll need some token in  order to attest a claim.   
```

Comment:
Wordy: "You'll need tokens to attest a claim." just removed the some and in order and added an 's' to tokens.

It is a huge improvement. 💯